demo/inferno-router-demo/tsconfig.json

```diff
@@ -0,0 +1,18 @@
+{
+  "compilerOptions": {
+    "esModuleInterop": true,
```

Comment:
it would be nice to have these 2 settings false: 

```
    "esModuleInterop": false,
    "isolatedModules": false,
``` 
but then code needs some cleaning

Reply:
WILL INVESTIGATE

- I believe I had an issue with third-party libraries, but I will give it a try.

Reply:
Fixed in 12d29a3